languages/en.ini

```diff
@@ -258,6 +259,11 @@ date_year_placeholder = "Y"
 Debug Information = "Debug Information"
 del_search = "Remove Search Group"
 Delete = "Delete"
+delete_account_confirm = "Are you sure you want to delete your account?"
+delete_account_description_html = "Saved searches and favorite lists will be deleted. You will still be able to login again later to create a new account."
```

Comment:
Not sure if "login again later" makes sense here -- once their account is deleted, they may or may not be able to log in again later, depending on the authentication method used. (External auth methods will let them log in again, but database auth will require creation of a new account; perhaps we should make this language, and any translations of it, a little more generic -- maybe say something like "you can still establish a new account later if you wish").